src/test/scala/com/codacy/parsers/CloverParserTest.scala

```diff
@@ -0,0 +1,108 @@
+package com.codacy.parsers
+
+import java.io.File
+
+import com.codacy.parsers.implementation.CloverParser
+import org.scalatest.{BeforeAndAfterAll, EitherValues, Matchers, WordSpec}
+
+class CloverParserTest extends WordSpec with BeforeAndAfterAll with Matchers with EitherValues {
+  private val CoberturaReportPath = "src/test/resources/test_cobertura.xml"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Field name 'CoberturaReportPath' does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://app.codacy.com/gh/codacy/coverage-parser/pullRequest?prid=4898850)